bin/main.py

```diff
@@ -57,14 +57,22 @@ def run_file(file_path):
     print('Starting tool execution...')
     config = load_config(CONFIG_PATH)
     #TODO decide how to give these parameters
-    intervals = range(config['test_param']['minInterval'], config['test_param']['maxInterval']+config['test_param']['intervalStep'], 
+    intervals = range(config['test_param']['minInterval'], config['test_param']['maxInterval']+config['test_param']['intervalStep'],
         config['test_param']['intervalStep'])
-    gasLimit = range(config['test_param']['minGas'], config['test_param']['maxGas']+config['test_param']['gasStep'], 
+    gasLimit = range(config['test_param']['minGas'], config['test_param']['maxGas']+config['test_param']['gasStep'],
         config['test_param']['gasStep'])
-    
+
+    node_number = config['eth_param']['nodeNumber']
+
+    for i in range(1, node_number + 1):
```

Comment:
This must not be done in the main script, if anything is done in the SUT deployment script or we loose modularity.

Reply:
Yeah, you are right. I didn't think about it.